src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -2,19 +2,46 @@ import React, { useState } from 'react';
 import { FormattedMessage, useIntl } from 'react-intl';
 import { Helmet } from 'react-helmet';
 import SubscriberHistoryCurrentSearch from './SubscriberHistoryCurrentSearch/SubscriberHistoryCurrentSearch';
+import { Formik, Form, Field } from 'formik';
 
 const SubscriberHistory = () => {
   const intl = useIntl();
   const [state, setState] = useState({ subscriberSearch: '' });
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
 
-  const searchSubscriber = () => {
-    setState({ showResults: true, subscriberEmail: state.subscriberSearch });
+  const fieldNames = {
+    search: 'search',
   };
 
-  const handeChange = (event) => {
-    setState({ showResults: false, subscriberSearch: event.target.value });
+  const getFormInitialValues = () => {
+    const values = Object.keys(fieldNames).reduce(
+      (accumulator, currentValue) => ({ ...accumulator, [currentValue]: '' }),
+      {},
+    );
+
+    return values;
+  };
+
+  const validateSearch = (value) => {
+    let error;
+    if (!value) {
+      error = 'Required';
+    } else if (value.length < 3) {
```

Comment:
We could set this 3 as part of a constant in utils, to generalize this criteria for all searches. What do you think? Or add this into a const in a file with a meaningful name.

Reply:
ok thanks, you right

Reply:
ready